index.js

```diff
@@ -52,6 +53,7 @@ module.exports = function(config) {
     (!config.cloudwatchNamespace && config.cloudwatchStackname)) throw new Error('both cloudwatchNamespace and cloudwatchStackname must be configured');
 
   config.maxProcessTime = config.maxProcessTime || 3e5;
+  config.minProcessTime = config.minProcessTime || 0;
```

Comment:
Some projects that use kine already have their own timers. Will forcing a minimum timeout > 0 in kine cause issues in those projects?